src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderStreamsTest.cs

```diff
@@ -0,0 +1,611 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests
+{
+    public static class DataReaderStreamsTest
```

Comment:
Please also consider including tests that use Async command APIs for command execution and reading, i.e. `ExecuteReaderAsync`, `ReadAsync`, etc. since Async is our sensitive flow area.

Reply:
Contributions welcome.

Reply:
Sure, I'll make a PR to your branch for this one to ease this one.

Reply:
Please take a look: https://github.com/Wraith2/SqlClient/pull/2